### ISSUE TITLE
[SIEM][Filebeat] Palo Alto Networks module: Remove zero NAT addresses

### ISSUE THREAD

**Issue by adriansr** [elastic/beats#13331]
Palo Alto Networks logs seem to always include a value for `{source,destination}.nat.{ip,port}` fields, even when no NAT is performed. In that case the fields are zero values: `0.0.0.0` and `0` port.

Those must be removed.

```
        "destination.nat.ip": "0.0.0.0",
        "destination.nat.port": 0,
```

as seen on #13330

**Comment by legoguy1000** [comment]
This looks to have been resolved by #18525 and can be closed @adriansr

**Comment by adriansr** [comment]
Right, @legoguy1000 

Closing.
